host/sgx/sgxsign.c

```diff
@@ -107,7 +107,6 @@ static oe_result_t _get_exponent(
     const oe_rsa_public_key_t* rsa,
     uint8_t exponent[OE_EXPONENT_SIZE])
 {
-    static const uint8_t OE_SGX_SIGNING_EXPONENT[] = {3, 0, 0, 0};
```

Comment:
This PR has a bad merge that undoes previous fixes, please ensure that this and line 130-133 are not removed

Reply:
Sure